collector/app/views.py

```diff
@@ -1,7 +1,57 @@
 """This module provides views for collector app."""
 
+from aiohttp import web
+from core.jwt import decode_jwt
 
-from aiohttp.web import RouteTableDef
+from app.utils.monobank import parse_transaction_response
 
 
-routes = RouteTableDef()
+routes = web.RouteTableDef()
+
+
+@routes.view("/monobank/{token}")
+class MonobankWebhook(web.View):
+    """Class that represent functionality to work with monobank webhook."""
+
+    def _check_permission(self):
+        """Return user_id if provided token is correct."""
+        token = self.request.match_info["token"]
+        secret = self.request.app["constants"]["MONOBANK_WEBHOOK_SECRET"]
+        decoded_token = decode_jwt(token, secret)
+        if decoded_token:
+            return None
+
+        user_id = decoded_token.get("user_id")
+        return user_id
+
+    async def post(self):
+        """Receive transaction by monobank webhook and save transaction."""
+        user_id = self._check_permission()
+        if not user_id:
```

Comment:
Maybe also make it with the exception. But actually, we should discuss which approach we should use further.

Reply:
Yeap, make sense. I've read somewhere: better to catch except than use if condition. In this case core should be updated, I don't want to do this for now. Let's make the new approach in server application/